src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -385,6 +385,8 @@ protected void OnPropertyChanged(string propertyName)
 
         public bool IsPackageDeprecated => _packageMetadata?.DeprecationMetadata != null;
 
+        public bool IsDeprecationControlVisible => IsPackageDeprecated && _searchResultPackage.PackageLevel == PackageLevel.TopLevel;
```

Comment:
If we are now only loading metadata for top-level packages, then the vulnerability and deprecation data should not be there for this PackageItemViewModel and so IsPackageDeprecated and IsPackageVulnerable should be false, right? I'm not seeing why we'd need another boolean that checks again that the packages are top-level but I am probably missing something. Can you help me understand why we need the extra boolean for visibility?

Reply:
It's just an easy fix to hide deprecation an vulnerability metadata. I now see you point on disabling metadata in details pane: I can not remove the network metadata call,  but I can stop 'calculating' vulnerability results in code.

Reply:
Locally installed packages are supposed to get their metadata from the installed packages, right? Is that not happening for installed packages? Does it call to the registration API to get metadata?

Can we change how transitive dependencies are getting their metadata and not make network calls?